cmd/api/app/routes/clusteroverridepolicy/handler.go

```diff
@@ -91,48 +91,6 @@ func handlePostClusterOverridePolicy(c *gin.Context) {
 	common.Success(c, "ok")
 }
 
-func handlePutClusterOverridePolicy(c *gin.Context) {
```

Comment:
update clusteroverridepolicy and clusterpropagationpolicy have been unified in UnstructedObject api, it should be deleted 👍